lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -271,4 +280,84 @@ class StandardAccountTest {
         val receipt2 = provider.getTransactionReceipt(result2.transactionHash).send()
         assertEquals(TransactionStatus.ACCEPTED_ON_L2, receipt2.status)
     }
+
+    @Test
+    fun `estimate deploy account fee`() {
+        val provider = gatewayProvider
```

Comment:
Why not use `if (provider !is GatewayProvider) return` like I suggested? Once we want to extend this test test suite (which is likely with this test suite), we'll have to write more code here instead of just removing one line.

Reply:
Because it overcomplicates a simple case. Why should I use parametrised test if I can run only one case? If we ever need to change that we will change that. Changes would be required in both cases, so I prefer to keep it simpler.

Reply:
All other tests in this module are parametrized so it's inconsistent.

Reply:
Tests should be easy to read, doing it this way will make it more complex. Just because other tests use require some parameters doesn't mean that every test needs that.

Reply:
You're trading ease to to maintain the test suite for the ease to read (arguably) and I'm not sure if that's a good thing.

Reply:
I disagree that it makes maintenance of the tests harder, I think it makes is easier. Parametrised tests make sense when they actually parametrise something, otherwise they just obfuscate the real logic that is used. 

@THenry14 please provide your feedback here, I'll do what you decide.

Reply:
I guess it doesn't really matter which way we go, BUT imho tests should be as simple as possible and test only what they are supposed to test + returning nothing actually tests nothing. Moreover, it can be parametrized when actually needed. So, sorry Artur but 
<img width="483" alt="obraz" src="https://user-images.githubusercontent.com/9470712/200818325-be8b7a89-2d77-4566-b651-b092f081f144.png">

Reply:
😭

Reply:
Also we agreed with Artur to add FIXME comment before the test.